### PR TITLE
add values to parameters copied from symfony docs

### PR DESCRIPTION
**Pull request by pscheit**

### CODE REVIEW

**Comment by indeyets** [comment]
values, which are not `null` should be in quotes

File: doc/providers/swiftmailer.rst [hunk 1]
```diff
@@ -21,8 +21,8 @@ Parameters
   * **port**: SMTP port, defaults to 25.
   * **username**: SMTP username, defaults to an empty string.
   * **password**: SMTP password, defaults to an empty string.
-  * **encryption**: SMTP encryption, defaults to null.
-  * **auth_mode**: SMTP authentication mode, defaults to null.
+  * **encryption**: SMTP encryption, defaults to null.  Valid values are 'tls', 'ssl', or null (indicating no encryption).
```
Comment (GromNaN) [inline doc/providers/swiftmailer.rst]: You have 2 spaces.

**Comment by pscheit** [comment]
well sorry, if this litte thing is such an act to mergen. Just to improve a really small part of the documentation I will think twice about the next pull request

**Comment by pscheit** [comment]
I can't imagine how difficult it has to be to send an actual pull request that CHANGES something

**Comment by indeyets** [comment]
@pscheit thank you for your contribution. merges to large projects take time. fabien is a busy person and we just try to make sure, that this fix wouldn't require more fixes at later stage. you did excellent job and this patch can be merged now as soon as @fabpot reaches silex's pr-queue

**Comment by fabpot** [comment]
Thank you @pscheit.

**Comment by pscheit** [comment]
okay, well sorry. This was a ruff day, yesterday..

best regards
Philipp

**Pull request closed without merging**